packages/kit/src/core/sync/write_server.js

```diff
@@ -0,0 +1,89 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { posixify, resolve_entry } from '../../utils/filesystem.js';
+import { s } from '../../utils/misc.js';
+import { load_error_page, load_template } from '../config/index.js';
+import { runtime_directory } from '../utils.js';
+
+/**
+ * @param {{
+ *   hooks: string | null;
+ *   config: import('types').ValidatedConfig;
+ *   has_service_worker: boolean;
+ *   runtime_directory: string;
+ *   template: string;
+ *   error_page: string;
+ * }} opts
+ */
+const server_template = ({
+	config,
+	hooks,
+	has_service_worker,
+	runtime_directory,
+	template,
+	error_page
+}) => `
+import root from './root.svelte';
+import { set_building, set_paths, set_version } from '${runtime_directory}/shared.js';
+
+set_paths(${s(config.kit.paths)});
+set_version(${s(config.kit.version.name)});
+
+export const options = {
+	csp: ${s(config.kit.csp)},
+	csrf_check_origin: ${s(config.kit.csrf.checkOrigin)},
+	embedded: ${config.kit.embedded},
+	env_public_prefix: '${config.kit.env.publicPrefix}',
+	hooks: null, // added lazily, via \`get_hooks\`
+	root,
+	service_worker: ${has_service_worker},
+	templates: {
+		app: ({ head, body, assets, nonce }) => ${s(template)
+			.replace('%sveltekit.head%', '" + head + "')
+			.replace('%sveltekit.body%', '" + body + "')
+			.replace(/%sveltekit\.assets%/g, '" + assets + "')
+			.replace(/%sveltekit\.nonce%/g, '" + nonce + "')},
```

Comment:
Previously in file `packages/kit/src/exports/vite/dev/index.js` the body and head replaces were after nonce and assets, just in case someone attempted to reference them within compiled svelte code. Should that still be the case here?

Reply:
It makes no difference — no replacement is happening at runtime